src/pages/Login/Login.js

```diff
@@ -1,8 +1,91 @@
-import React, { Component } from 'react';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Login.scss';
+
+class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      userId: '',
+      userPw: '',
+    };
+  }
+
+  // 입력 함수
+  handleChange = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  // Back이랑 연결하는 fetch 함수
+  requestLogin = () => {
+    fetch('http://10.58.5.217:8000/users/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.userId,
+        password: this.state.userPw,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.token) {
+          alert('로그인 성공');
+          localStorage.setItem('access_token', result.token);
+        } else {
+          alert('로그인 실패');
+        }
+      });
+  };
 
-export class Login extends Component {
   render() {
-    return <div>Login</div>;
+    const { userId, userPw } = this.state;
+    return (
+      <div className="Login">
+        <div className="loginView">
+          <div className="logoWrapper">
+            <img
+              alt="logo"
+              className="loginLogo"
+              src="http://localhost:3000/images/logo.png"
```

Comment:
```suggestion
              src="/images/logo.png"
```
http://localhost:3000 기입하실 필요 없습니다!